cpp/src/gandiva/gdv_function_stubs.h

```diff
@@ -120,6 +120,9 @@ const char* gdv_fn_sha1_decimal128(int64_t context, int64_t x_high, uint64_t x_l
                                    int32_t x_precision, int32_t x_scale,
                                    gdv_boolean x_isvalid, int32_t* out_length);
 
+int gdv_fn_utctime_to_zone(int* time_fields, const char* zone, int zone_len,
```

Comment:
I looked for that function inside the gdv_stubs, but I did not find it

Reply:
removed it